.travis.yml

```diff
@@ -13,34 +13,17 @@ php:
 matrix:
   fast_finish: true
 
-env:
-  matrix:
-    - DEPENDENCIES="high"
-    - DEPENDENCIES="low"
-  global:
-    - DEFAULT_COMPOSER_FLAGS="--no-interaction --no-ansi --no-progress --no-suggest"
-
 before_install:
   - ./tools/composer clear-cache
 
 install:
-  - if [[ "$DEPENDENCIES" = 'high' ]]; then travis_retry ./tools/composer update $DEFAULT_COMPOSER_FLAGS; fi
-  - if [[ "$DEPENDENCIES" = 'low' ]]; then travis_retry ./tools/composer update $DEFAULT_COMPOSER_FLAGS --prefer-lowest; fi
-
-before_script:
-  - echo 'zend.assertions=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - echo 'assert.exception=On' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - composer update --no-ansi --no-interaction --no-progress --no-suggest
 
 script:
-  - ./phpunit --coverage-clover=coverage.xml
-  - ./phpunit --configuration ./build/travis-ci-fail.xml > /dev/null; if [ $? -eq 0 ]; then echo "SHOULD FAIL"; false; else echo "fail checked"; fi;
   - xmllint --noout --schema phpunit.xsd phpunit.xml
```

Comment:
Do I understand correctly that after this has been merged, Travis would still be used but only for the `xmllint` checks? We should be able to also do that with a GitHub action so that we can completely abandon Travis, right?

Reply:
Will follow-up!

Reply:
I could also solve this here, though, if you prefer, @sebastianbergmann!

Reply:
What do you mean by "here"?

Reply:
@sebastianbergmann 

Here, in this pull request!

Reply:
Sure! :)